src/components/Recipes/RecipePage.tsx

```diff
@@ -0,0 +1,35 @@
+import { useParams } from "react-router-dom";
+import React, { useEffect, useState } from "react";
+import "./RecipePage.css"
+import { readRecipe } from "../../utils/SpoonacularRequests"
+import RecipeInfo from "../../models/recipeInfo";
+
+const RecipePage: React.FC = () => {
+    const {recipeID} = useParams();
+    const [recipeData, setRecipeData] = useState<RecipeInfo | null>(null);
```

Comment:
we are using use useReducer/useContext so we need to keep state in consistent placees

Reply:
only basic state can be added within in a component if it only peertains to that component -- something more crucial to the app should be "globalized"